lib/service/cache/redis.js

```diff
@@ -208,6 +208,22 @@ class Redis extends Service {
     return this._client.mget(_args);
   }
 
+  /**
+   * Executes multiple client commands in a single action
+   *
+   * @returns {Promise}
+   */
+  mexecute (commands) {
+    if (!Array.isArray(commands) || commands.length === 0) {
+      return Bluebird.resolve([]);
+    }
+
+    return new Bluebird(async (resolve) => {
+      const pipeline = this._client.multi(commands);
+      resolve(pipeline.exec());
+    });
+  }
```

Comment:
`pipeline.exec` returns a Promise from what I understand, so this should work while being much simpler:

```suggestion
  mexecute (commands) {
    if (!Array.isArray(commands) || commands.length === 0) {
      return Bluebird.resolve([]);
    }

    return this._client
      .multi(commands)
      .exec();
  }
```